.github/workflows/build.yml

```diff
@@ -583,8 +584,20 @@ jobs:
           node-version: ${{ needs.setup-build.outputs.node-version }}
           os: ${{ runner.os }}
 
+      - name: Setup PDM
+        uses: pdm-project/setup-pdm@568ddd69406b30de1774ec0044b73ae06e716aa4 # v4.1
+        with:
+          python-version-file: .python-version
+          version: ${{ env.PDM_VERSION }}
```

Comment:
Shouldn't the action pull the version from the dot file?

Reply:
which dot file? we've no dot file for pdm version

Reply:
Sry it has been late. I mixed up the python and PDM version.